doc/src/stdlib/linalg.md

```diff
@@ -73,6 +73,10 @@ Base.LinAlg.diagm
 Base.LinAlg.scale!
 Base.LinAlg.rank
 Base.LinAlg.norm
+Base.LinAlg.norm(::AbstractVector, ::Real)
```

Comment:
Yeah, it would make sense to include all methods, and not only the function. I can't imagine a situation where you would like to include the docstring for the function, but not for the methods.